oracle/project/oracle-node/.env.example

```diff
@@ -1,4 +1,4 @@
 API_URL="https://min-api.cryptocompare.com/data/price?fsym=ETH&tsyms=USD&api_key=<your api key here>"
-ORACLE_CONTRACT_ID=0x1bc1332bff931c3cfc9752fe10fe30e1ed3c41f77101f7d144c8183355e5590e
+ORACLE_CONTRACT_ID=0x3926c54eb171e0d0bb7921b15e96fd08e53bd1cbd5244e2d0d997b045811779f
```

Comment:
ID changed, is it because its source code is different?

Reply:
Exactly. Added a note in the README for developers and users to double check that this ID matches the deployed contract's ID.